docs/en-US/component/tree-v2.md

```diff
@@ -116,6 +116,8 @@ tree-v2/filter
 | expandNode | expand specified node | `(node: TreeNode)` |
 | collapseNode | collapse specified node | `(node: TreeNode)` |
 | setData | When the data is very large, using reactive data will cause the poor performance, so we provide a way to avoid this situation | `(data: TreeData)` |
+| scrollTo | scroll to a given position | `(offset: number)` |
```

Comment:
Please add a version number, like this `scrollTo ^(2.8.0)`
```suggestion
| scrollTo ^(2.8.0) | scroll to a given position | `(offset: number)` |
```

Reply:
updated